rpcserver.go

```diff
@@ -1337,6 +1337,7 @@ func (r *rpcServer) GetTransactions(context.Context,
 			Amount:           tx.Value.ToBTC(),
```

Comment:
Think you could amend this PR slightly to modify the protos to instead display `satoshis` fro the `TransactionDetails` struct? 

A follow up PR could be made that modifies the `WalletBalance` RPC to also return satoshis instead of BTC. 

I think with those two modifications, all external facing RPC will now purely deal with satoshis.